femwell/examples/overlap_integrals.py

```diff
@@ -15,8 +15,16 @@
 
 for width in widths:
     with tempfile.TemporaryDirectory() as tmpdirname:
-        mesh = mesh_waveguide(wsim=2, hclad=.7, hbox=.5, wcore=width, hcore=0.22, filename=tmpdirname + '/mesh.msh')
-        mesh = Mesh.load(tmpdirname + '/mesh.msh')
+        mesh = mesh_waveguide(
+            wsim=2,
+            hclad=0.7,
+            hbox=0.5,
+            wcore=width,
+            hcore=0.22,
+            filename=f'{tmpdirname}/mesh.msh',
+        )
+
```

Comment:
```suggestion
mesh = mesh_waveguide(wsim=2, hclad=.7, hbox=.5, wcore=width, hcore=0.22, filename=f'{tmpdirname}/mesh.msh')
```
I'd guess breaking the parameters into lines would lower the readability, as the file focuses on implementing the simulation and the parameters for the example are not in focus here. The usage of f-strings is nice!